docs/tutorials/developer-journey/level-1/1.3-first-dapp.md

```diff
@@ -0,0 +1,723 @@
+        margin-top: 0;
+      }
+
+      h2 {
+        font-size: 24px;
+        margin-bottom: 10px;
+        text-align: center;
+      }
+
+      form {
+        margin-bottom: 20px;
+        border: 2px solid #8bc34a;
+        padding: 20px;
+        border-radius: 5px;
+      }
+
+      label {
+        display: block;
+        margin-bottom: 10px;
+        font-size: 18px;
+        text-align: left;
+      }
+
+      input[type="radio"] {
+        margin-right: 5px;
+      }
+
+      button {
+        padding: 10px 20px;
+        background-color: #007bff;
+        border: none;
+        color: #fff;
+        font-size: 18px;
+        cursor: pointer;
+        border-radius: 5px;
+      }
+
+      button#reset {
+        background-color: #dc3545;
+        position: absolute;
+        bottom: 20px;
+        left: 20px;
+        margin: 10px 0; /* Add margin */
+      }
+
+      button:hover {
+        background-color: #0056b3;
+      }
+
+      #results {
+        margin-top: 20px;
+        font-size: 18px;
+        border: 2px solid #8bc34a;
+        padding: 20px;
+        border-radius: 5px;
+        position: relative;
+      }
+    </style>
+</head>
+<body>
+    <div class="container">
+      <div class="title-container">
+        <h1>Simple Voting Poll</h1>
+      </div>
+      <h2 id="question">Sample Question</h2>
+
+      <!-- Form where users vote -->
+      <div class="form-container">
+        <form id="radioForm">
+            <label>
+                <input type="radio" name="option" value="Rust"> Rust
+            </label><br>
+            <label>
+                <input type="radio" name="option" value="Motoko"> Motoko
+            </label><br>
+            <label>
+                <input type="radio" name="option" value="TypeScript"> TypeScript
+            </label><br>
+            <label>
+                <input type="radio" name="option" value="Python"> Python
+            </label><br>
+            <button type="submit">Vote</button>
+        </form>
+      </div>
+
+      <!-- Poll results appear here-->
+      <h2 id="results-title">Results</h2>
+      <div id="results"></div>
+    </div>
+    <button id="reset">Reset Poll</button>
+
+
+</body>
+</html>
+```
+
+:::info
+#### What does this code do?
+The HTML code above is a simple form that provides the end user with options to select using radio buttons. There is nothing 'Web3' or specific to the IC about this code.
+
+Then, the `<head>` tag is used to include some basic CSS for the page's styling. To learn more about adding a stylesheet, see: [add a stylesheet](/docs/developer-docs/frontend/add-stylesheet.md).
+:::
+
+Lastly, we need to update the `index.js` file in order for our frontend canister to communicate with our backend canister and have the result of that communication reflected within the frontend's HTML. 
+
+Open the `/src/poll_frontend/src/index.js` file in your code editor and replace the content of `index.js` with the following:
+
+
+```javascript
```

Comment:
as a general comment - I do think that in this day and age, vanilla JS may be more difficult to parse for new developers than say, React. many developers are introduced to the world of programming through tools like React; so much so that vanilla JS may appear daunting.

of course the trade off is that if you used a React app as the basis of the tutorial, you either have to explain the addition of all the libraries or hand-wave through all that stuff.